front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -224,10 +229,33 @@ export default function AssistantNew({
                   setShowAllAgents(!showAllAgents);
                 }}
               />
-              <StartHelperConversationButton
-                content="Hey @helper, how do I use the assistant?"
-                handleSubmit={handleSubmit}
-              />
+              {isBuilder ? (
+                <>
+                  <Button
+                    variant="primary"
+                    icon={Cog6ToothIcon}
+                    label="Manage Assistants"
+                    onClick={() => {
+                      void router.push(`/w/${owner.sId}/builder/assistants`);
+                    }}
+                  />
+                  <Button
```

Comment:
The three buttons layout is responsive?

Reply:
<img width="619" alt="Capture d’écran 2023-09-22 à 16 28 41" src="https://github.com/dust-tt/dust/assets/3803406/5c2b2c06-a8f4-4376-a355-8a97743fa4fa">
🙇🏻‍♀️

Reply:
yay